### DISCUSSION TITLE
v0.5.1

### DISCUSSION THREAD

**Discussion by SoCuul** [SoCuul/SCInsta#90] [Announcements]
> Tested on Instagram v349.1.11
> Media downloading features are currently being rewritten! Expect to see them fully working in a future release.

## New Features:
- Notify user when padlock authentication fails 98f6bff920531a415bfaaab44299c7aa4287a99b
- Improve confirm change theme functionality to work with new theme picker 6fd7725a00bdafe0dc9c928aca5e2d842cf0df4d

## Fixes:
- Fix confirm like not always working cf95975e7fa3c60fa322dd79fc059f73c3490b72

## Other:
- You can now build SCInsta with the use of GitHub Actions (thanks to @zeuikli)! Visit the [Building IPA page](https://github.com/SoCuul/SCInsta/wiki/Building-IPA) to learn more.

## New Contributors
* @zeuikli made their first contribution in https://github.com/SoCuul/SCInsta/pull/28

---

**Full Changelog**: https://github.com/SoCuul/SCInsta/compare/v0.5.0...v0.5.1

<hr /><em>This discussion was created from the release <a href='https://github.com/SoCuul/SCInsta/releases/tag/v0.5.1'>v0.5.1</a>.</em>

**Comment by IamNickerson** [comment]
LETS GO GITHUB ACTIONS

**Reply by IamNickerson** [reply]
This ass tho instructions are fucking terrible

**Reply by SoCuul** [reply]
I gave step by step instructions haha, what part is confusing?
